data/remnant missions.txt

```diff
@@ -1252,3 +1252,214 @@ mission "Remnant: Expanded Horizons Quarg 3"
+	on offer
+		conversation
+			`You have salvaged another broken jump drive, and the Remnant seemed eager to pay you for them. Would you like to offer them this one?`
+			choice
+				`	(No.)`
+					defer
+				`	(Yes.)`
+			`	You stop by the Outfitter, and discover that the engineer there has placed a standing offer for broken jump drives. "We are always eager to have more" he sings enthusiastically.`
+			choice
+				`	"Is there any progress on the research I could hear about?"`
+				`	"Excellent, I will continue to bring any others I find."`
+					accept
+			`	The engineer checks his terminal, and after skimming a bit he looks up again. "Well, it looks like there are several different teams working on this. The one that has updated their most recent report is looking into the similarities between standard hyperdrives and these broken jump drives. As you might expect, a device like these should not suddenly change functionality to perfectly match a completely different sort of device just because it was damaged. The researchers think that this down-stepping failure might be significant in understanding what jump drives even are."`
+			`	The engineer continues to share excerpts from various research team's reports with you, and it becomes increasingly clear that while it appears that the Remnant are making significant discoveries in several different directions, they are probably a very long way from anything definitive.`
+				accept
+	
+	on accept
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+		fail "Remnant: Broken Jump Drive 2"
+
+
+
+mission "Remnant: Broken Jump Drive 4"
+	name "More Jump Drives"
+	minor
+	invisible
+	landing
+	to offer
+		"remnant: broken jump drive count" == 3
+	to complete
+		has "Jump Drive (Broken)"
+	source
+		attributes "remnant"
+	destination 
+		attributes "remnant"
+	on offer
+		conversation
+			`You have salvaged another broken jump drive and brought it back to Remnant territory. Do you want to take advantage of their standing offer?`
+			choice
+				`	(No.)`
+					defer
+				`	(Yes.)`
+			`	You stop by the Outfitter, and notify an engineer that you have a drive for them. A few minutes later a young engineer is leading a team onto the <ship> to retrieve the malfunctioning drive. She hands you your payment and thanks you for supporting their research.`
+			choice
+				`	"Is there any news from the researchers?"`
+				`	"Your welcome."`
+					accept
+			`	The engineer checks her terminal, and jumpts through a few pages before turning back to you. "A new update just came in today!" She trills excitedly. "One team has made some progress investigating one of the mechanism, and it seems to add extra dimensions to the small pocket of regular space that the drive occupies. They suspect that this multi-dimensional function enables it to store significantly more equipment in the same space, which may contribute to their unstablity."`
```

Comment:
"Unstability" to "Instability", "Unstability" is not a real word.

Reply:
`jumpts` -> `jumps`
`one of the mechanism` -> `one of the mechanisms`